manifests/golang.yml

```diff
@@ -468,6 +468,18 @@ tests/:
     test_otel_sdk_interoperability.py: missing_feature
     test_otel_span_with_baggage.py:
       Test_Otel_Span_With_Baggage: missing_feature
+    test_parametric_endpoints.py:
+      Test_Parametric_DDSpan_Add_Link: missing_feature (add_link endpoint is not implemented)
+      Test_Parametric_DDSpan_Set_Resource: missing_feature (set_resource endpoint is not implemented)
+      Test_Parametric_DDTrace_Baggage: missing_feature (baggage is not supported)
```

Comment:
Baggage _is_ supported (you can add a baggage item to a span with `span.SetBaggageItem(k, v)`, but yes the parametric doesn't have an endpoint that allows you to set baggage.
Also, I'm not sure what you mean by "baggage is not supported" -- like, baggage is supported, but not up to the otel spec. So what does it mean for baggage to be supported?

Reply:
Ahh that makes sense. Yes I meant doesn't support baggage according to the RFC that Lucas wrote. I will update this comment to be more descriptive.